otter/scheduler.py

```diff
@@ -19,81 +19,47 @@
 from otter.util.hashkey import generate_transaction_id
 
 
-class SchedulerService(TimerService):
+class SchedulerService(MultiService, object):
```

Comment:
Why is this `MultiService`? We only appear to be using `setServiceParent`, not `addService`.